powerauth-push-server/src/main/resources/application.properties

```diff
@@ -53,6 +53,8 @@ powerauth.push.service.registration.multipleActivations.enabled=false
 powerauth.push.service.registration.retry.backoff=100
 powerauth.push.service.registration.retry.maxAttempts=2
 
+powerauth.push.service.clients.cache.expireAfterWrite=24h
```

Comment:
@zcgandcomp What is the reasonable default?

Reply:
It seems that 24 hours is too long, but 5 minutes is too short because initializing the client is quite heavy. Let's consider some smart reloadable cache.